server/db.js

```diff
@@ -31,9 +31,19 @@ const addUser = async (_user) => {
 const addPic = async (url, id) => {
     try {
         // Fetch user with the given id
-        const user = await Users.findById(id);
+        const user = await Users.findById(id).populate("conversations");
+
         //add url to user document
-        user.imageUrl = url
+        user.imageUrl = url;
+
+        //add url to conversation documents
+        const conversations = user.conversations;
+        conversations.forEach(function (conversation) {
+            conversation.imageUrl = url;
+            conversation.save();
```

Comment:
when a user updates their picture, the update is done to each picture stored in their conversations